manifests/cli.pp

```diff
@@ -31,16 +31,6 @@
   # The jenkins cli command with required parameter(s)
   $cmd = "java -jar ${jar} -s http://localhost:${port}"
 
-  # Reload all Jenkins config from disk (only when notified)
-  exec { 'reload-jenkins':
```

Comment:
There's an explanation in the commit message: https://github.com/jhoblitt/puppet-jenkins/commit/06c50ce14e93ec787acd77c84e76590aaecf2830